src/API/Management/Connections.php

```diff
@@ -1,122 +1,187 @@
+        // Pagination.
+        if (null !== $page) {
+            $params['page'] = abs(intval($page));
+            if (null !== $per_page) {
+                $params['per_page'] = $per_page;
+            }
         }
 
-        return $request->call();
+        return $this->apiClient->method('get')
+            ->addPath('connections')
+            ->withDictParams($params)
+            ->call();
     }
 
     /**
-     * @param string $id
-     * @param null|string|array $fields
-     * @param null|string|array $include_fields
-     * @return mixed
+     * Get a single Connection by ID.
+     *
+     * @param string $id - Connection ID to get.
+     * @param null|string|array $fields - Fields to include or exclude from the result, empty to retrieve all fields.
+     * @param null|boolean $include_fields - True to include $fields, false to exclude $fields.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/get_connections_by_id
      */
-    public function get($id, $fields = null, $include_fields = null) 
+    public function get($id, $fields = null, $include_fields = null)
     {
-        $request = $this->apiClient->get()
-            ->connections($id);
+        $params = [];
 
-        if ($fields !== null) 
-        {
-            if (is_array($fields)) 
-            {
-                $fields = implode(',', $fields);
+        // Results fields.
+        if (!empty($fields)) {
+            $params['fields'] = is_array($fields) ? implode(',', $fields) : $fields;
+            if (null !== $include_fields) {
+                $params['include_fields'] = $include_fields;
             }
-            $request->withParam('fields', $fields);
-        }
-
-        if ($include_fields !== null) 
-        {
-            $request->withParam('include_fields', $include_fields);
         }
 
-        return $request->call();
+        return $this->apiClient->method('get')
+            ->addPath('connections', $id)
+            ->withDictParams($params)
+            ->call();
     }
 
     /**
-     * @param string $id
-     * @return mixed
+     * Delete a Connection by ID.
+     *
+     * @param string $id - Connection ID to delete.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/delete_connections_by_id
      */
-    public function delete($id) 
+    public function delete($id)
     {
-        return $this->apiClient->delete()
-            ->connections($id)
+        return $this->apiClient->method('delete')
+            ->addPath('connections', $id)
             ->call();
     }
 
     /**
-     * @param string $id
-     * @param string $email
-     * @return mixed
+     * Delete a specific User for a Connection.
+     *
+     * @param string $id - Connection ID (currently only database connections are supported).
```

Comment:
AKA `auth0` strategies. Is there another way to rephrase this database only warning?